nin/backend/nin

```diff
@@ -63,6 +64,29 @@ program
     generate.generate(projectRoot, resource, name);
   });
 
+program
+  .command('update')
+  .description('Update nin to newest version')
+  .action(function() {
+    echo('Checking for new version of nin');
+
+    var ninExecutable = __dirname;
+    var ninPath = ninExecutable.split('/');
+    var ninPath = ninPath.splice(0, ninPath.length-3).join('/');
```

Comment:
Ok, we can't use `__dirname` in that case. On my computer it returns `/Users/cristea/Christoffersting/Projects/nin/nin/backend`. The code here normalizes the path by removing the last `nin` and `backend`.